test/gtest/parser_unroll.cpp

```diff
@@ -235,7 +232,12 @@ std::pair<const IR::P4Parser*, const IR::P4Parser*> rewriteParser(const IR::P4Pr
 
 /// Loads example from a file
 const IR::P4Program* load_model(const char* curFile, CompilerOptions& options) {
-    setenv("P4C_16_INCLUDE_PATH", "../p4include", 1);
+    const char* sourceDir = getenv("P4C_SOURCE_DIR");
+    BUG_CHECK(sourceDir, "Can't find environment variale 'P4C_SOURCE_DIR'"); 
+    std::string relPath = sourceDir;
+    relPath.append("/");
```

Comment:
`relPath` at this point points to `<p4c>/`. This means that the `includeDir` is just fine, but `options.file` points for example to `<p4c>/parser-unroll-test1.p4` not to the `<p4c>/testdata/p4_16_samples/parser-unroll-test1.p4`. Or even worse to `<p4c>/../p4_14_samples/switch_20160512/switch.p4` in case of the `switch_20160512` test. I am guessing the full paths within the p4c root directory need to be added for all of the `loadExample()` calls.

Reply:
Yes, you are right, thanks...